src/components/leadbox/leadbox.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Leadbox } from './leadbox';
+import { Astronaut } from '../icons/illustrations/astronaut';
+
+export default {
+  component: Leadbox,
+  title: 'Components/Leadbox',
+  parameters: {
+    layout: 'fullscreen',
+  },
+} as ComponentMeta<typeof Leadbox>;
```

Comment:
Nice that you added the `fullscreen` layout 👌 
When I tested this in storybook I desperately wanted to change the icon so I had a look at [the documentation](https://storybook.js.org/docs/react/essentials/controls#dealing-with-complex-values) and it turns out we can have arbitrary values in the controls. With some more configuration we would be able to turn the `icon` into a control as well:
```suggestion
import { Astronaut } from '../icons/illustrations/astronaut';
import { SpaceShuttle } from '../icons/illustrations/space-shuttle';

const icons = { Astronaut: (<Astronaut />), SpaceShuttle: (<SpaceShuttle />) };

export default {
  component: Leadbox,
  title: 'Components/Leadbox',
  parameters: {
    layout: 'fullscreen',
  },
  argTypes: {
    icon: {
      options: Object.keys(icons),
      mapping: icons,
    }
  }
} as ComponentMeta<typeof Leadbox>;
```